package.json

```diff
@@ -43,7 +43,8 @@
     "prepare-patch-release": "run-s all version:patch doc:publish",
     "prepare-minor-release": "run-s all version:minor doc:publish",
     "prepare-major-release": "run-s all version:major doc:publish",
-    "prepack": "npm ci && npm run build"
+    "prepack": "npm ci && npm run build",
+    "prepare": "npm run build"
```

Comment:
Why add this script? You'll have to call it with `npm run prepare`, but at that point you can still run `npm run build`

Reply:
Prepare script is added there to have ability install the SDK specific version from the git branch and have it built. Without it npm is not triggering build, when I install the specific version in dc-cli project.
This will be removed after release.

Reply:
Ah, I see. That makes sense.